getting-started.md

```diff
@@ -0,0 +1,299 @@
+in the account the credentials were created for, which we will revisit as the last step to this guide. For now, we will start by navigating to the `Files` page.
+
+## Files
+
+The `Files` page within the Vertex Developer Dashboard allows for basic operations on Vertex Platform Files ([API Reference]((https://docs.vertex3d.com/#83fe0cae-da2d-4e3f-9c66-258bba1116ca))). Through the `Files` page, you can:
+* View existing files
+  * Includes both files created through the dashboard as well as files created through our APIs directly
+* Create new files
+* Delete existing files
+* Inspect file status and properties
+
+### Creating a new File
+
+From this view, the `New` button in the top right corner of the list can be used to create Files and upload data to them. The file creation dialog provides
+a set of optional parameters that can be configured to suit a variety of different use cases.
+
+<details>
+  <summary>New File Example</summary>
+
+  ![Create File](./public/list-and-create-file.png)
+</details>
+<br />
+
+* `Supplied ID` - refers to a user-provided string that can be used to uniquely identify a file. Setting this is useful if you intend to reference these files
+from an external application to avoid the need to store the auto-generated `ID` of the file. Note that this is globally unique across the platform account, and
+no two files can share the same supplied ID.
+* `Root File Name` - indicates the name of the root model file within an assembly file. For non-CAD files, or non-assembly files, this field should be left blank.
+* `Create Part After Upload` - indicates whether we intend to create a part within the `Parts Library` with the data uploaded. If checked, this file will be uploaded 
+to the Vertex Platform and you will automatically navigate to the `Parts Library` tab to complete part creation.
+
+### Creating a new Part
+
+Once `Upload` is pressed, you will be auto-navigated to the `Parts Library` page, where we can complete the Part creation. The part creation dialog provides 
+a set of inputs to provide `Supplied ID`s, similar to the file creation dialog.
+
+
+<details>
+  <summary>New Part Example</summary>
+
+  ![Create Part](./public/create-part.png)
+</details>
+<br />
+
+* `Supplied ID` - refers to a user-provided string that can be used to uniquely identify a part. Similar to the `Supplied ID` for files, this value is globally unique
+across the platform account, and no two parts can share the same supplied ID.
+* `Supplied Revision ID` - refers to a user-provided string that can be used to uniquely identify a part revision. Contrary the `Supplied ID`, the `Supplied Revision ID`
+is not required to be globally unique within the platform account. When combined with the part `Supplied ID`, this ID is expected to be globally unique, so a part
+with a `Supplied ID` of `part-a` could not have two revisions with a `Supplied Revision ID` of `rev-1` for instance.
```

Comment:
What do you think about 
`...not required to be globally unique within the platform account. However, this ID must be unique for a given part.  For instance, a part with a Supplied ID of part-a could not have two revisions with a Supplied Revision ID of rev-1.`?

Reply:
Yeah, I like this wording better - updated!